### ISSUE TITLE
Not able to initiate dance without removing first

### ISSUE THREAD

**Issue by LucyMcGowan** [jhudsl/matahari#20]
This code is creating an issue where you need to run `dance_remove()` in order to run `dance_start()` again (which I don't think we intended?) before you could just run 
```
dance_start()
dance_stop()
```
https://github.com/jhudsl/matahari/blob/834885d06dd3496b6741d4a5932d5e35173e878c/R/dance.R#L40-L42

**Comment by seankross** [comment]
This should be fixed in #21. Would you mind testing it out for me?

```
remotes::install_github("jhudsl/matahari@dev")
```
```

**Comment by LucyMcGowan** [comment]
Fabulous, just tried it and it looks good :+1: Thank you, @seankross you're the best!
